client/go/dogma/cmd/cmd.go

```diff
@@ -0,0 +1,271 @@
+// Copyright 2018 LINE Corporation
+//
+// LINE Corporation licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+// Package cmd implements commands that execute HTTP API to download, add, create, modify, or delete the
+// specified resources on the Central Dogma server.
+// Commands are created according to the input from the CLI and executed.
+package cmd
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/line/centraldogma/client/go/dogma"
+	"github.com/urfave/cli"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+// Command is the common interface implemented by all commands.
+type Command interface {
+	// execute executes the command
+	execute(c *cli.Context) error
+}
+
+func getRemoteURL(remoteURL string) (string, error) {
+	if len(remoteURL) == 0 {
+		fmt.Print("Enter specific address to connect: ")
+		scanner := bufio.NewScanner(os.Stdin)
+		if !scanner.Scan() {
+			return "", errors.New("you must input specific address")
+		}
+		line := strings.TrimSpace(scanner.Text())
+		if len(line) == 0 {
+			return "", errors.New("you must input specific address")
```

Comment:
How about performing some validation, and replacing `you must input specific address` with: `Invalid server address`?

Reply:
Golang is saying that `Error string should not be capatilized ...`, so let me do `invalid server address`. Just telling you that I don't want to make you surprised. :-)